test/libdnf/rpm/test_package_set.hpp

```diff
@@ -0,0 +1,72 @@
+/*
+Copyright (C) 2020 Red Hat, Inc.
+
+This file is part of libdnf: https://github.com/rpm-software-management/libdnf/
+
+Libdnf is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+Libdnf is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+
+#ifndef TEST_LIBDNF_RPM_PACKAGE_SET_HPP
+#define TEST_LIBDNF_RPM_PACKAGE_SET_HPP
+
+
+#include "libdnf/rpm/package_set.hpp"
+#include "libdnf/rpm/sack.hpp"
+
+#include <cppunit/TestCase.h>
+#include <cppunit/extensions/HelperMacros.h>
+
+
+class RpmPackageSetTest : public CppUnit::TestCase {
+    CPPUNIT_TEST_SUITE(RpmPackageSetTest);
+
+    #ifndef WITH_PERFORMANCE_TESTS
+    CPPUNIT_TEST(test_add);
+    CPPUNIT_TEST(test_contains);
+    CPPUNIT_TEST(test_remove);
+//    CPPUNIT_TEST(test_union);
+//    CPPUNIT_TEST(test_intersection);
+//    CPPUNIT_TEST(test_difference);
```

Comment:
Why did you commented it? Please could you mark it as todo with an explanation or remove it?

Reply:
All 3 tests have been implemented.